### DISCUSSION TITLE
How do I close the coroutine I started with cocotb.start_soon

### DISCUSSION THREAD

**Discussion by tearorca** [cocotb/cocotb#4228] [Q&A] [answered]
Hello，How do I close the coroutine I started with cocotb.start_soon? I saw in the official manual that the related functions are deprecated. Thank you for your answer

**Comment by ktbarrett** [comment, marked as answer]
[`Task.kill`](https://docs.cocotb.org/en/stable/library_reference.html#cocotb.task.Task.kill).
